php/tests/GeneratedClassTest.php

```diff
@@ -1902,6 +1913,18 @@ public function testNoSegfaultWithError()
         new TestMessage(['optional_int32' => $this->throwIntendedException()]);
     }
 
+    public function testNoSegfaultWithContructorBypass() {
```

Comment:
CS: the other functions in this class place their opening parenthesis on a new line
```suggestion
    public function testNoSegfaultWithContructorBypass()
    {
```